nowy.js

```diff
@@ -0,0 +1,198 @@
+        leadingMinutes = "0" + minutes.toString();
+      } else {
+        leadingMinutes = minutes;
+      }
+      if(hours < 10){
+        leadingHours = "0" + hours.toString();
+      } else {
+        leadingHours = hours;
+      } 
+ displayTimer = document.querySelector('.timer').innerText = leadingHours + ':' + leadingMinutes + ':' + leadingSeconds + ":" + milliseconds;
+}
+
+const resetBtn = document.getElementById('reset-button');
+const splitBtn = document.getElementById('split-button');
+
+
+const startActionBtn = document.getElementById('start-button');
+
+
+
+const startBtn = document.querySelector('.startOrPause');
+const pauseBtn = document.getElementById('pause-button'); 
+startBtn.addEventListener('click', startTimer);
+pauseBtn.addEventListener('click', startTimer);
+
+function startTimer () {
+    if (timerStatus === "stopped") {
+    timerInterval = window.setInterval(stopWatch, 10);
+    document.getElementById('start-button').style.display = 'none';
+    document.getElementById('pause-button').style.display = 'block';
+    document.getElementById('reset-button').style.display = 'none';
+    document.getElementById('split-button').style.display = 'block';
+    timerStatus = "started"
+
+} else {
+
+    window.clearInterval(timerInterval);
+    document.getElementById('start-button').style.display = 'block';
+    document.getElementById('pause-button').style.display = 'none';
+    document.getElementById('reset-button').style.display = 'block';
+    document.getElementById('split-button').style.display = 'none';
+    timerStatus = "stopped";
+}};
+
+
+resetBtn.addEventListener('click', timeAndDataReset);
+
+function timeAndDataReset() {
+    milliseconds = 0;
+    seconds = 0;
+    hours = 0;
+    minutes = 0;
+    displayTimer = document.querySelector('.timer').innerText = '00:00:00:00';
+
+    document.querySelector('.lp-num').innerHTML = '';
+    document.querySelector('.split-results').innerHTML = '';
+    document.querySelector('.total-time').innerHTML = '';
+    LapNumber = 0;
+    totalTimes = [];
+    intervalTimes = [];
+  };
+
+let totalTimes = [];
+let intervalTimes = [];
+let LapNumber = 0;
+
+const timeString = document.querySelector('.timer').innerText;
+
+function parseTimeToMilliseconds(timeString) {
+ 
+  const parts = timeString.split(':');
+  const hours = parseInt(parts[0]);
+  const minutes = parseInt(parts[1]);
+  const seconds = parseInt(parts[2]);
+  const milliseconds = parseInt(parts[3]);
+  
+  const totalMilliseconds = hours * 3600000 + minutes * 60000 + seconds * 1000 + milliseconds;
+  return totalMilliseconds;
+}
+function millisecondsToTimeFormat(milliseconds) {
+  const hours = Math.floor(milliseconds / 3600000);
+  milliseconds %= 3600000;
+  const minutes = Math.floor(milliseconds / 60000);
+  milliseconds %= 60000;
+  const seconds = Math.floor(milliseconds / 1000);
+  const remainingMilliseconds = milliseconds % 1000;
+
+  return (
+    hours.toString().padStart(2, '0') + ':' +
```

Comment:
Create a separate function `padTime(timeValue, padCount)` and use it here. For `hours`, `minutes`, `seconds` padCount will be 2. For `miliseconds` 3